### ISSUE TITLE
Support for Motorola G5

### ISSUE THREAD

**Issue by Marco85Li** [nielsbasjes/yauaa#111]
Hi team

I found the following user agent coming from Firefox Focus installed on Moto G5 Plus (not G5S Plus).

`Mozilla/5.0 (Linux; Android 7.0; Moto G (5) Plus Build/NPNS25.137-93-14) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Mobile Safari/537.36`

This instance has been reported as coming from Chrome 69.0.3497.100 on a Generic Android device.

Is it possible to include the new device in the next release?

Marco

**Comment by nielsbasjes** [comment]
Hi,

I added this useragent to the current version and it is currently reported as below.
It says 'Chrome 69' because there is absolutely no indication of Firefox Focus present in the useragent string you provided.

```
- test:
    input:
      user_agent_string: 'Mozilla/5.0 (Linux; Android 7.0; Moto G (5) Plus Build/NPNS25.137-93-14) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Mobile Safari/537.36'
    expected:
      DeviceClass                          : 'Phone'
      DeviceName                           : 'Motorola Moto G (5)'
      DeviceBrand                          : 'Motorola'
      OperatingSystemClass                 : 'Mobile'
      OperatingSystemName                  : 'Android'
      OperatingSystemVersion               : '7.0'
      OperatingSystemNameVersion           : 'Android 7.0'
      OperatingSystemVersionBuild          : 'NPNS25.137-93-14'
      LayoutEngineClass                    : 'Browser'
      LayoutEngineName                     : 'Blink'
      LayoutEngineVersion                  : '69.0'
      LayoutEngineVersionMajor             : '69'
      LayoutEngineNameVersion              : 'Blink 69.0'
      LayoutEngineNameVersionMajor         : 'Blink 69'
      AgentClass                           : 'Browser'
      AgentName                            : 'Chrome'
      AgentVersion                         : '69.0.3497.100'
      AgentVersionMajor                    : '69'
      AgentNameVersion                     : 'Chrome 69.0.3497.100'
      AgentNameVersionMajor                : 'Chrome 69'

```

**Comment by Marco85Li** [comment]
Perfect. I'll close the issue.

**Comment by nielsbasjes** [comment]
Just now I released version 5.3 which includes this.
